steam/scripts/app_build_342980.vdf

```diff
@@ -2,8 +2,8 @@
 {
 	"appid"	"342980"
 	"desc" "Your build description here" // description for this build
-	"buildoutput" "..\output\" // build output folder for .log, .csm & .csd files, relative to location of this file
-	"contentroot" "..\content\DestinationSol\" // root content folder, relative to location of this file
+	"buildoutput" "output\" // build output folder for .log, .csm & .csd files, relative to location of this file
+	"contentroot" "..\..\desktop\build\distributions\DestinationSol\" // root content folder, relative to location of this file
```

Comment:
If you change the "DestinationSol" directory here to "app" then you shouldn't need to change the output the directory path in the gradle scripts.

Reply:
Yeah, the reason for that though was so you could download a Jenkins build and just extract it right in. The folder is ‘DestinationSol’ in Jenkins.

Reply:
Right, I misunderstood that. 

I don't think it's a good idea to unzip the file to the same directory path as where you would build a distribution locally as it introduces a risk that you may upload a local build rather than the build from Jenkins.